test/test.pmon/README

```diff
@@ -0,0 +1,4 @@
+short test demonstrating power management over sleeps. Sleeps are 5, 10, 15 
+seconds. the most deeply-nested regions should have energy scaled to the length
+of the sleep. Outer regions should be the sum of the inner regions. Power consumption
+is low because of the sleeps.
```

Comment:
Can you add example IPM output from test.pmon to the README file highlighting which values should be 2x/3x larger?